### DISCUSSION TITLE
[FEAT] Output in SVG

### DISCUSSION THREAD

**Discussion by Kochise** [csgoh/processpiper#14] [Ideas]
Hi, is there a way to output the diagram in SVG format ?

Regards.

**Comment by csgoh** [comment]
Hi, Processpiper is relying on Pillow (PIL) package to generate the diagram. Unfortunately, PIL cannot output to SVG. PIL is a raster image processor, and SVG is a vector image format. Raster images are composed of pixels, while vector images are composed of shapes and paths.

Do you mind sharing why you need the output in SVG format?

**Comment by Kochise** [comment]
Better resolution and smaller file size.

**Comment by macaroni57** [comment]
I understand that SVG is not on the roadmap. Some other advantages are embedding links into a node or edge, and ability to customize offline (though defeats the purpose of the tool).

**Comment by csgoh** [comment]
@macaroni57 SVG will be supported in the next release. 😉

**Comment by csgoh** [comment]
@macaroni57, @Kochise 
FYI, v0.5.0 has just been released. You can now save diagram in SVG format. See the [Wiki HOW-TO](https://github.com/csgoh/processpiper/wiki/How-To#how-to-save-diagram-in-svg) page for more details.

**Comment by Kochise** [comment]
@csgoh Thank, I'll review this asap (didn't received the notification).
